apps/src/music/views/MusicView.jsx

```diff
@@ -58,10 +65,12 @@ class UnconnectedMusicView extends React.Component {
   static propTypes = {
     appOptions: PropTypes.object,
     appConfig: PropTypes.object,
-    levels: PropTypes.array,
-    currentLevelIndex: PropTypes.number,
+    /** If Music Lab is being rendered as a standalone lab (i.e. under /projectbeats) */
+    standalone: PropTypes.bool,
```

Comment:
Will this boolean be confusing when we add a `/projects/musiclab/` route? We often refer to those as "standalone" projects.

Reply:
Hmm, good point. Maybe 'incubator' since this version is accessed via the incubator page?

Reply:
yeah that makes sense!